android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainViewModel.kt

```diff
@@ -338,6 +336,10 @@ constructor(
     }
   }
 
+  fun dataRefreshLivedataPostValue(refresh: Boolean) {
+    dataRefreshLivedata.postValue(refresh)
+  }
```

Comment:
why add this?

Reply:
This was for testing purposes.
The code and test have been updated.